rules/do-you-use-gitignore-for-your-repo/rule.md

```diff
@@ -1,8 +1,8 @@
 ---
-seoDescription: Learn why and how to use `.gitignore` to manage clean repositories. Understand key patterns and explore gitignore templates for different projects.
+seoDescription: Learn why and how to use .gitignore to manage clean repositories. Understand key patterns and explore gitignore templates for different projects.
 type: rule
-title: Do you use a `.gitignore` file to keep your Git repository clean?
-uri: use-gitignore-for-clean-repo
+title: Do you use a .gitignore file to keep your Git repository clean?
+uri: do-you-use-gitignore-for-clean-repo
```

Comment:
I'm not sure if `title` and `uri` need to be exactly the same.

1. Could you confirm with @bradystroud if what you have is okay?

Reply:
Might be hard to get brady on CTF day, i asked @JackDevAU instead 
![image](https://github.com/user-attachments/assets/a23f26e5-560e-4fd9-bb98-7be858bc4572)